File: Program8.py
#	Write a program which accepts number from user and print all odd numbers upto that number

def displayOddNumbers(iNum):
	
	for iCnt in range(0,int(iNum/2),1):
		
		if((iCnt % 2) == 1):
			
			print(iCnt,end = ' ')
			
	print()	
	

def main():

	print("Enter number : ")
	iNo = int(input())
	
	displayOddNumbers(iNo)

if __name__ == "__main__":
	main()

File: Program8.cs
/*
	Write a program which accepts number from user and print all odd numbers upto that number
*/
using System;

class Numbers
{
	private int iNo;
	
	public Numbers(int iNum)
	{
		iNo = iNum;
	}
	
	public void displayOddNumbers()
	{
		int iCnt = 0;
		
		for(iCnt = 1; iCnt < iNo/2; iCnt++)
		{
			if((iCnt % 2) == 1)
			{
				Console.Write(iCnt+"\t");
			}
		}
		Console.WriteLine();
	
	}
	

}

class EntryPointMain
{

	static int Main()
	{
		int iNo = 0;
		
		Console.WriteLine("Enter number : ");
		iNo = Convert.ToInt32(Console.ReadLine());
		
		Numbers nObj = new Numbers(iNo);
		
		nObj.displayOddNumbers();
		
		return 0;	
	}

}
